### ISSUE TITLE
Error: Unbound module CL.Response

### ISSUE THREAD

**Issue by djs55** [mirage/ocaml-github#90]
My [travis job](https://travis-ci.org/avsm/opam-boot/jobs/85132446) hit this:

```
=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
∗  installed cmdliner.0.9.8
∗  installed ocamlfind.1.5.5
∗  installed base-bytes.base
∗  installed base64.2.0.0
∗  installed camomile.0.8.5
∗  installed cppo.1.3.1
∗  installed easy-format.1.0.2
∗  installed biniou.1.0.9
∗  installed magic-mime.1.0.0
∗  installed camlp4.4.02+6
∗  installed menhir.20151012
∗  installed ocplib-endian.0.8
∗  installed atd.1.1.2
∗  installed ppx_tools.0.99.3
∗  installed react.1.2.0
∗  installed re.1.4.1
∗  installed ssl.0.5.0
∗  installed stringext.1.4.0
∗  installed type_conv.113.00.02
∗  installed fieldslib.113.00.00
∗  installed lwt.2.5.0
∗  installed sexplib.113.00.00
∗  installed yojson.1.2.3
∗  installed cstruct.1.7.0
∗  installed atdgen.1.7.1
∗  installed ipaddr.2.6.1
∗  installed uri.1.9.1
∗  installed zed.1.4
∗  installed conduit.0.8.8
∗  installed lambda-term.1.9
∗  installed cohttp.0.19.3
[ERROR] The compilation of github failed at "ocaml setup.ml -build".
...
# cd lib && atdgen -j -j-std github.atd
# /home/travis/build/avsm/opam-boot/_obj/opamroot/4.02.3/bin/ocamlfind ocamldep -package atdgen -package cohttp -package cohttp.lwt-core -package lwt -package uri -package yojson -modules lib/github_j.mli > lib/github_j.mli.depends
# /home/travis/build/avsm/opam-boot/_obj/opamroot/4.02.3/bin/ocamlfind ocamlc -c -g -annot -bin-annot -principal -strict-sequence -package atdgen -package cohttp -package cohttp.lwt-core -package lwt -package uri -package yojson -w @5@8@10@11@12@14@23@24@26@29 -I lib -o lib/github_j.cmi lib/github_j.mli
# /home/travis/build/avsm/opam-boot/_obj/opamroot/4.02.3/bin/ocamlfind ocamldep -package atdgen -package cohttp -package cohttp.lwt-core -package lwt -package uri -package yojson -modules lib/github_j.ml > lib/github_j.ml.depends
# /home/travis/build/avsm/opam-boot/_obj/opamroot/4.02.3/bin/ocamlfind ocamldep -package atdgen -package cohttp -package cohttp.lwt-core -package lwt -package uri -package yojson -modules lib/github_t.ml > lib/github_t.ml.depends
# /home/travis/build/avsm/opam-boot/_obj/opamroot/4.02.3/bin/ocamlfind ocamlc -c -g -annot -bin-annot -principal -strict-sequence -package atdgen -package cohttp -package cohttp.lwt-core -package lwt -package uri -package yojson -w @5@8@10@11@12@14@23@24@26@29 -I lib -o lib/github_core.cmo lib/github_core.ml
# + /home/travis/build/avsm/opam-boot/_obj/opamroot/4.02.3/bin/ocamlfind ocamlc -c -g -annot -bin-annot -principal -strict-sequence -package atdgen -package cohttp -package cohttp.lwt-core -package lwt -package uri -package yojson -w @5@8@10@11@12@14@23@24@26@29 -I lib -o lib/github_core.cmo lib/github_core.ml
# File "lib/github_core.ml", line 314, characters 20-33:
# Error: Unbound module CL.Response
# Command exited with code 2.
```

**Comment by dsheets** [comment]
It's a constraint problem in the opam metadata. I'm on it.

**Comment by dsheets** [comment]
Ah, you are using an old version of ocaml-github <https://github.com/avsm/opam-boot/blob/master/.travis-ci.sh#L28>. The metadata (and code) is fine.

**Comment by djs55** [comment]
Ah! avsm/ocaml-github indeed. Sorry for the noise!

On Tue, Oct 13, 2015 at 4:50 PM, David Sheets <notifications@github.com>
wrote:

> Closed #90 <https://github.com/mirage/ocaml-github/issues/90>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mirage/ocaml-github/issues/90#event-434188634>.
>



-- 
Dave Scott
